content/en/docs/contribute/style/style-guide.md

```diff
@@ -38,29 +38,25 @@ The English-language documentation uses U.S. English spelling and grammar.
 
 {{< comment >}}[If you're localizing this page, you can omit the point about US English.]{{< /comment >}}
 
-## Documentation content standards
+## What Kubernetes documentation does and doesn't contain
 
-Kubernetes documentation does not host certain types of content:
+Kubernetes documentation only hosts content from projects within the
+ `kubernetes` or `kubernetes-sigs` GitHub organizations or from current CNCF
+  projects.
 
-* Vendor, provider, or project-specific content from projects that are not:
-
-    * Inside the kubernetes or kubernetes-sigs GitHub organizations
-    * Inside the CNCF
-
-* Identical content hosted in multiple locations ("dual-sourced" content)
+Kubernetes documentation does not host identical content sourced from
+ multiple locations ("dual-sourced" content).
```

Comment:
nit: If the content is not identical is the content still considered `dual-sourced`?

Reply:
@kbhawkey Based on my conversations with @zacharysarah, I'd say yes. I changed "identical" to "duplicate"  - not much difference I know.